ui/templates/welcome.html

```diff
@@ -3,7 +3,11 @@
     <div class="section-header">
         <div class="row">
             <div class="col-md-2 col-md-offset-5">{% if perms.learningresources.add_repository %}<a href="{% url 'create_repo' %}" role="button"
-                                                     class="btn btn-primary btn-large">Create repository</a>{% endif %}</div>
+                                                     class="btn btn-primary btn-large">Create repository</a>
```

Comment:
made a couple of tests with the `{% spaceless %}` tag and it does not solve the problem of an extra space in front of the string in the button.
So I'm going to merge it as it is